icic_extraction/grid.py

```diff
@@ -48,21 +61,59 @@ def extract_transaction_from_summary_table (summary_table):
     summary_table_data = []
     for transaction  in summary_table:
         clean_transaction = tuple(element.strip() for element in transaction)
```

Comment:
cleaning should be done on the first element only, a loop is not necessary

Reply:
I think inner loop is necessary

Reply:
what does a transaction represent here?
is it a list of summary items(buy,sell,total,comapny)?

Reply:
list of tuples